ceph-container-build-push-imgs/config/definitions/ceph-container-build-push-imgs.yml

```diff
@@ -39,3 +39,8 @@
       - inject-passwords:
```

Comment:
should we remove these lines too?

Reply:
That's a good idea. Let's do that in a separate PR